Tests/FixtureIntegrationTest.php

```diff
@@ -364,7 +364,12 @@ protected function configureRoutes(RouteCollectionBuilder $routes) : void
 
     protected function configureContainer(ContainerBuilder $c, LoaderInterface $loader) : void
     {
-        $c->setParameter('kernel.secret', 'foo');
+        $c->loadFromExtension('framework', [
+            'secret' => 'foo',
```

Comment:
The `FrameworkExtension` of FrameworkBundle will set the `kernel.secret` parameter.